### PR TITLE
Mbd

### PR DESCRIPTION
**Pull request by mchiu-bnl**
[comment]: <> (Please tell us something about this pull request)
Added support for using CDB for calibrations

## Types of changes
[comment]: <> ( What types of changes does your code introduce? Put an `x` in all the boxes that apply: )
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work for users)
- [ ] Requiring change in macros repository (Please provide links to the macros pull request in the last section)
- [x] I am a member of [GitHub organization of sPHENIX Collaboration](https://github.com/orgs/sPHENIX-Collaboration/people), EIC, or ECCE (contact Chris Pinkenburg to join)

## What kind of change does this PR introduce? (Bug fix, feature, ...)

[comment]: <> ( What does this PR do? Linking to talk in software meeting encouraged )

## TODOs (if applicable)

[comment]: <> ( In case this is a draft PR, e.g. for running checks using Jenkins, please make the pull request as a draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/  )

### CODE REVIEW

**Comment by pinkenburg** [comment]
The QA failure is a jenkins issue (it ran out of threads)

File: offline/packages/bbc/Makefile.am
Comment (pinkenburg) [inline offline/packages/bbc/Makefile.am:1]: the dependency of libbbc_io.so  libffamodules will break online, this needs to be removed

File: offline/packages/bbc/BbcDefs.h
Comment (pinkenburg) [inline offline/packages/bbc/BbcDefs.h:1]: please use c from gsl

File: offline/packages/bbc/BbcCalib.cc
Comment (pinkenburg) [inline offline/packages/bbc/BbcCalib.cc:1]: no singletons

File: offline/packages/bbc/BbcDefs.h
Comment (mchiu-bnl) [reply offline/packages/bbc/BbcDefs.h:1]: Done.

File: offline/packages/bbc/BbcCalib.cc
Comment (mchiu-bnl) [reply offline/packages/bbc/BbcCalib.cc:1]: Done

File: offline/packages/bbc/BbcDefs.h
Comment (mchiu-bnl) [reply offline/packages/bbc/BbcDefs.h:1]: Done

File: offline/packages/bbc/Makefile.am
Comment (mchiu-bnl) [reply offline/packages/bbc/Makefile.am:1]: Done.

**Pull request merged**